AlayaCare Cloud Tech Radar.csv

```diff
@@ -45,3 +46,5 @@
 "Hold","Rancher","Tools","FALSE","<p></p>"
 "Hold","require.js","languages-and-frameworks","FALSE","<p></p>"
 "Hold","single-spa","languages-and-frameworks","FALSE","<p><a href='https://single-spa.js.org/'>single-spa</a> is a JavaScript framework for bringing together multiple micro frontends in a single front-end application.</p>"
+"Hold","Jenkins","Tools","FALSE","<p><a href='https://www.jenkins.io/'>jenkins.io</a></p>"
+"Hold","Zoom","Tools","FALSE","<p><a href='https://alayacare.zoom.us/'>Virtual-meetings</a></p>"
```

Comment:
Question:
I might lack the latest freshest context, but isn't it premature to move Zoom to Hold while we are still evaluating it? Last public update made was 9 days ago: https://alayacare.slack.com/archives/C02GV1R3Y/p1707595223316319?thread_ts=1707512662.024039&cid=C02GV1R3Y

Reply:
This is no longer true for Zoom